package/gluon-mesh-vpn-tunneldigger/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/lua
+
+function restartTunneldigger()
+	io.popen('logger -t tunneldigger-watchdog "Restarting Tunneldigger."')
+	os.execute("/etc/init.d/tunneldigger restart")
+end
+
+function readPidFile()
+	local pidFile = io.open("/var/run/tunneldigger.mesh-vpn.pid", "r")
+	local returnPid = "0"
+	if pidFile~=nil then 
+		returnPid = pidFile:read("*l") 
+		io.close(pidFile)
+	end
+	return returnPid
+end
+
+function getNeighbourCount()
+        local list = io.lines("/sys/kernel/debug/batman_adv/bat0/originators")
+        if list~=nil then
+                for line in list do
+                        if line:find('mesh%-vpn') then
+                                return 1
+                        end
+                end
+        end
+        return nil
+end
+
+local uci = require('simple-uci').cursor()
+local tunneldiggerEnabled = uci:get_bool('tunneldigger', 'mesh_vpn', 'enabled') 
+if tunneldiggerEnabled then
```

Comment:
there is no point to bind that name if you are only using it once.

Reply:
There is, self explaining coding conventions to make code easily readable without comments. The only reason not to do this would be to save some space.

Reply:
I changed this. Let's not discuss about it :)